public/content/documentation/web/component/number-input.md

```diff
@@ -0,0 +1,219 @@
+   - Number keys: Numbers are entered
+   - Non-number keys: Nothing is entered
+
+2. Test mobile screenreader gestures
+
+   - Swipe: Focus moves to the input, numeric keypad is revealed
+
+3. Listen to screenreader output on all devices
+
+   - Name: Its purpose is clear
+   - Role: It identifies itself as an editable input
+   - Group: Hints or errors are read after the label, related inputs include a group name (Ex: Enter your personal information)
+   - State: If applicable, it expresses its state (required, disabled / dimmed / unavailable)
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/number-input](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/number-input)
+
+## Gherkin
+
+### #a11y - Web Accessibility Acceptance Criteria
+
+How to test a number input
+
+GIVEN THAT I am on a page with a number input field
+
+1. Keyboard for mobile & desktop
+
+   - WHEN I use the tab key to move focus to a number input I SEE focus is strongly visually indicated
+   - THEN when I use the number keys I SEE numbers are entered
+   - THEN when I use non-number keys I SEE nothing is entered
+
+2. Desktop screenreader
+
+   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) 
+      - AND I use the tab key to move focus to a number input
+      - I HEAR its purpose is clear
+      - I HEAR it identifies itself as an editable input
+      - I HEAR hints or errors are read after the label, related inputs include a group name (Ex: Enter your personal information)
+      - I HEAR if applicable, it expresses its state (required, disabled / dimmed / unavailable)
+   - THEN when I use the number keys I HEAR numbers are entered
+   - THEN when I use non-number keys I HEAR nothing is entered
+
+3. Mobile screenreader
+
+   - WHEN I use a mobile screenreader (Talkback, VoiceOver)
+      - AND I swipe to focus on a number input
+      - I HEAR its purpose is clear
+      - I HEAR it identifies itself as an editable input
+      - I HEAR hints or errors are read after the label, related inputs include a group name (Ex: Enter your personal information)
+      - I HEAR if applicable, it expresses its state (required, disabled / dimmed / unavailable)
+   - THEN when I enter a number I HEAR the numeric keypad is revealed
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/number-input](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/number-input)
+
+## Code examples
+
+### Don't use `type="number"`
+
+The `type="number"` input is intended for **integers** and includes features we _don't_ want (like stepper/scroll functionality) that is a nuisance to everyone 
+
+- Phone, credit card, pin etc. are not integers
+- NVDA doesn't fully support `type="number"` inputs at this time
+
+### Use `type="text"` for number inputs
+
+Use `type=text` with `inputmode="numeric"` with an input pattern and JS to filter out non-numeric characters.
```

Comment:
I know this was on the original but I wonder if it could be beneficial to spell out "JavaScript" here instead of "JS" just to be clearer.

Reply:
Not a huge deal. Up to you.